README.md

```diff
@@ -502,529 +456,392 @@ Additionally, the computation may return a nullary function containing
-
-```hs
-encaseN  :: ((a,       Nodeback e r) -> x) -> a ->           Future e r
-encaseN2 :: ((a, b,    Nodeback e r) -> x) -> a -> b ->      Future e r
-encaseN3 :: ((a, b, c, Nodeback e r) -> x) -> a -> b -> c -> Future e r
-```
-
-</details>
-
-Allows [continuation-passing-style][1] functions to be turned into
-Future-returning functions.
-
-Takes a function which accepts as its last parameter a [Nodeback](#types), and
-a value, and returns a Future. When forked, the Future calls the function with
-the value and a Nodeback and resolves the second argument passed to the
-Nodeback, or or rejects with the first argument.
-
-Furthermore; `encaseN2` and `encaseN3` are binary and ternary versions of
-`encaseN`, applying two or three arguments to the given function respectively.
-
-```js
-var fs = require('fs');
-
-var read = Future.encaseN2(fs.readFile);
-
-read('README.md', 'utf8')
-.map(text => text.split('\n'))
-.map(lines => lines[0])
-.fork(console.error, console.log);
-//> "# [![Fluture](logo.png)](#butterfly)"
-```
-
-#### chainRec
-
-<details><summary><code>Future.chainRec :: ((a -> Next a, b -> Done b, a) -> Future e (Next a | Done b), a) -> Future e b</code></summary>
-
-```hs
-Future.chainRec :: ((a -> Next a, b -> Done b, a) -> Future e (Next a | Done b), a) -> Future e b
-```
-
-</details>
-
-Implementation of [Fantasy Land ChainRec][FL:chainrec]. Since Fluture 6.0
-introduced [stack safety](#stack-safety) there should be no need to use this
-function directly. Instead it's recommended to use [`chain(rec)`](#chain).
-
 ### Transforming Futures
 
 #### map
 
-<details><summary><code>map :: Functor m => (a -> b) -> m a -> m b</code></summary>
-
 ```hs
-map                  :: Functor m  => (a -> b) -> m a -> m        b
-Future.map           :: Functor m  => (a -> b) -> m a -> m        b
-Par.map              :: Functor m  => (a -> b) -> m a -> m        b
-Future.prototype.map :: Future e a ~> (a -> b)        -> Future e b
+map :: Functor m => (a -> b) -> m a -> m b
 ```
 
-</details>
-
-Transforms the resolution value inside the Future, and returns a Future with
-the new value. The transformation is only applied to the resolution branch: if
-the Future is rejected, the transformation is ignored.
+Transforms the resolution value inside the Future or [Functor][FL:functor],
+and returns a Future or Functor with the new value. The transformation is only
+applied to the resolution branch: if the Future is rejected, the transformation
+is ignored.
 
 See also [`chain`](#chain) and [`mapRej`](#maprej).
 
 ```js
-Future.of(1)
-.map(x => x + 1)
-.fork(console.error, console.log);
-//> 2
+> fork (log ('rejection'))
+.      (log ('resolution'))
+.      (map (x => x + 1) (resolve (41)))
+[resolution]: 42
 ```
 
 For comparison, the equivalent with Promises is:
 
 ```js
-Promise.resolve(1)
-.then(x => x + 1)
-.then(console.log, console.error);
+> Promise.resolve (41)
+.   .then (x => x + 1)
+.   .then (log ('resolution'), log ('rejection'))
```

Comment:
Why is this code now indented?

Reply:
I found that it reads easier because it creates a visual distinction between the statement and its return value / log lines. I typically don't like this style of indentation though.. :\

Reply:
I switched to the more consistent approach of not indenting anywhere.